src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -156,25 +170,9 @@ public void ActiveSolutionBoundTracker_Changes()
         {
             var boundProject = new BoundSonarQubeProject(new Uri("http://localhost:9000"), "key");
 
-            var solutionBinding = new ConfigurableSolutionBindingSerializer();
-
-            bool serviceIsConnected = false;
-            var mockSqService = new Mock<ISonarQubeService>();
-
-            Expression<Func<ISonarQubeService, Task>> connectMethod = x => x.ConnectAsync(It.IsAny<ConnectionInformation>(), It.IsAny<CancellationToken>());
-            Expression<Action<ISonarQubeService>> disconnectMethod = x => x.Disconnect();
+            SetupService(isConnected: false);
```

Comment:
I've added a couple of setup helper methods that hide some of the unpleasantness around setting up all of the mocks correctly.